docs/apidocs/ibm-runtime.rst

```diff
@@ -1,11 +1,10 @@
 *****************************************
-Qiskit IBM Quantum Provider API Reference
+Qiskit IBM Runtime API Reference
 *****************************************
 
 .. toctree::
    :maxdepth: 1
 
    ibm_runtime
    ibm_visualization
-   ibm_jupyter
-   ibm_utils
+   ibm_jupyter
```

Comment:
Side-note: why do we continue to need `ibm_visualization`, `ibm_jupyter`?

Reply:
Visualizing a backend can still be useful. But we can consider if we want to remove in a separate PR.